doc/conf.py

```diff
@@ -260,7 +261,10 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-# intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {
+    'http://docs.python.org/': None,
+    'https://docs.djangoproject.com/en/2.2': 'https://docs.djangoproject.com/en/2.2/_objects',
```

Comment:
Django 4.0 has entered feature freeze. Sure it shouldn't say at least "3.2"?

Reply:
(Also, isn't there a "latest"-url for Django-docs?)

Reply:
There is a latest-URL I could use, but I'm not sure I see the reason to link to the latest version as long as our dependency is locked to the LTS version 2.2 (other than the fact that we need to remember to change this link every time we change our deps)